edi_storage_oca/components/listener.py

```diff
@@ -30,30 +30,37 @@ def _add_after_commit_hook(self, move_func, sftp_filepath, sftp_destination_path
         )
 
     def on_edi_exchange_done(self, record):
-        storage = record.backend_id.storage_id
         res = False
-        if record.direction == "input" and storage:
-            file = record.exchange_filename
-            pending_dir = record.backend_id.input_dir_pending
-            done_dir = record.backend_id.input_dir_done
-            error_dir = record.backend_id.input_dir_error
-            if not done_dir:
-                return res
-            res = self._move_file(storage, pending_dir, done_dir, file)
+        backend = record.backend_id
```

Comment:
can you please review this so that we can merge it before?
https://github.com/OCA/edi/pull/556
It contains a small refactoring of this part.